From d5d45a1fd4e5269686ebd283e47cec5d1be2c46c Mon Sep 17 00:00:00 2001
From: Robert Ancell <robert.ancell@canonical.com>
Date: Wed, 27 Jun 2018 16:33:28 +1200
Subject: [PATCH 20/24] details page: Don't show missing screenshot placeholder

---
 src/gs-details-page.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gs-details-page.c b/src/gs-details-page.c
index c97450c3..d11b56a1 100644
--- a/src/gs-details-page.c
+++ b/src/gs-details-page.c
@@ -524,7 +524,7 @@ gs_details_page_refresh_screenshots (GsDetailsPage *self)
 		gtk_widget_set_visible (self->box_details_screenshot,
 		                        screenshots->len > 0);
 		gtk_widget_set_visible (self->box_details_screenshot_fallback,
-		                        screenshots->len == 0);
+		                        FALSE /*screenshots->len == 0*/);
 		return;
 	}
 
@@ -544,7 +544,7 @@ gs_details_page_refresh_screenshots (GsDetailsPage *self)
 		break;
 	default:
 		gtk_widget_set_visible (self->box_details_screenshot_fallback,
-					screenshots->len == 0);
+					FALSE /*screenshots->len == 0*/);
 		break;
 	}
 
-- 
2.20.1

